evm/src/cpu/kernel/asm/main.asm

```diff
@@ -84,7 +87,8 @@ execute_withdrawals_post_stack_op:
 global hash_final_tries:
     // stack: cum_gas, txn_counter, num_nibbles, txn_nb
     // Check that we end up with the correct `cum_gas`, `txn_nb` and bloom filter.
-    %mload_global_metadata(@GLOBAL_METADATA_BLOCK_GAS_USED_AFTER) %assert_eq
+    %mload_global_metadata(@GLOBAL_METADATA_BLOCK_GAS_USED_AFTER) 
+    %assert_eq
```

Comment:
Can you put it back in one line for consistency with all the others? Makes it clearer to bundle what's compared